fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -257,9 +253,6 @@ private Template getTemplate(final Model rdf, final Node subject,
             .map(x -> ((HtmlTemplate) x).value()).filter(templatesMap::containsKey).findFirst()
             .orElseGet(() -> {
                 final List<String> types = multiValueURI(rdf.getResource(subject.getURI()), type);
-                if (types.contains(REPOSITORY_NAMESPACE + "RepositoryRoot")) {
-                    return REPOSITORY_NAMESPACE + "RepositoryRoot";
```

Comment:
I suspect this was put in-place to prioritize the "RepositoryRoot" template... did your observations indicate otherwise?

Reply:
I did not do a lot of research into the reason for this. But `RepositoryRoot` is the first entry in the `templateMap` and so if this `types.contains()` is going to match then it should also match in the following `types.stream()...` and the `findFirst()` should return the first match, which should be `RepositoryRoot`.

Reply:
Now the test required me to re-add this. Remove it and it works, rebase and it fails. 🤷‍♂